accounts-db/src/accounts_db.rs

```diff
@@ -7407,6 +7407,22 @@ impl AccountsDb {
         (accounts_hash, total_lamports)
     }
 
+    /// Calculate the full accounts hash for `storages` and save the results at `slot`
```

Comment:
So are there other callers of `update_accounts_hash`? In the diff only the callers that changed to `update_accounts_hash_with_verify` show up. I guess it is confusing that there is a fn that was called `update_accounts_hash` that now does more than it used to and some callers call the old fn and some call the new one?